posix.mak

```diff
@@ -23,21 +37,40 @@ else
 	DOTLIB:=.a
 endif
 
-DFLAGS=$(MODEL_FLAG) -conf= -O -release -dip25 -inline -w -Isrc -Iimport $(PIC)
-UDFLAGS=$(MODEL_FLAG) -conf= -O -release -dip25 -w -Isrc -Iimport $(PIC)
 DDOCFLAGS=-conf= -c -w -o- -Isrc -Iimport -version=CoreDdoc
 
-CFLAGS=$(MODEL_FLAG) -O $(PIC)
+# Set CFLAGS
+CFLAGS=
+ifneq (,$(filter cc% gcc% clang% icc% egcc%, $(CC)))
```

Comment:
> So why did this work with your phobos build? You can still set CFLAGS explicitly.

I don't know, maybe the program has to reference the symbols in the .c files to trigger the linker error?

> You can still set CFLAGS explicitly.

Not a very workable solution as 1) I'd have to do it for every account / Digger instance 2) The required value of CFLAGS likely changes across D's history so it'd get in the way of bisecting.